tests/integrationtest/t/executor/simple.test

```diff
@@ -487,3 +487,30 @@ rollback;
 select * from auto_new;
 
 set autocommit = default;
+
+# TestDefaultAuthPluginForCreateUser
+
+connection default;
+set global default_authentication_plugin = 'caching_sha2_password';
+create user default_sha256_user;
```

Comment:
I think the `sha256` is confusing here as there used to be a `sha256_password` plugin.

```suggestion
create user default_sha2_user;
```
(more modifications needed if you merge this)

Reply:
updated in https://github.com/pingcap/tidb/commit/fa4a9a166f1a4a2c5b43602dab451f4c87d57ada